### PR TITLE
Add support for JCS Cryptosuites.

### PR DESCRIPTION
**Pull request by gannan08**

### CODE REVIEW

File: test/mocha/20-issue.js [hunk 1]
```diff
@@ -24,6 +24,16 @@ describe('issue', () => {
...
     },
+    'eddsa-jcs-2022': {
+      suiteName: 'eddsa-jcs-2022',
+      algorithm: 'Ed25519',
+      issueOptions: {},
+      statusOptions: {
+        suiteName: 'eddsa-jcs-2022',
+        algorithm: 'Ed25519'
+      },
+      tags: ['general']
```
Comment (dlongley) [inline test/mocha/20-issue.js:35]: I think we only need one suite to run through the general tests, no need to duplicate them -- or is there a good reason to add this one in particular?

```suggestion
      tags: []
```

**Review by dlongley — APPROVED** [review]
See suggestion.
